instrumentation/opentelemetry-instrumentation-asgi/src/opentelemetry/instrumentation/asgi/__init__.py

```diff
@@ -431,6 +431,19 @@ def get_host_port_url_tuple(scope):
     return server_host, port, http_url
 
 
+def maybe_inject_context(message, server_span, setter):
```

Comment:
I don't see much value in making this a helper function.

Reply:
It was either lint ignore too many branches (which I kind of agree with, the function had a lot of difficult to follow indentation) or this helper fn. Let me know which you would prefer and I will make the change.

Reply:
I'd add a lint ignore :)